api/ServiceWorkerGlobalScope.json

```diff
@@ -1295,7 +1258,7 @@
           "spec_url": "https://w3c.github.io/push-api/#dom-serviceworkerglobalscope-onpush",
           "support": {
             "chrome": {
-              "version_added": "40"
+              "version_added": "42"
```

Comment:
This now doesn't match Chrome for Android. Was it really shipped at different times on different platforms?

Reply:
I doubt this was added at different times.  The data came from the collector, so I'm thinking that it's possible Chrome 40 is incorrect.  I'd like to await a follow-up PR to look into this more.

Reply:
Would you mind splitting out the changes that make versions out of sync? I can look into them while reviewing, but if we merge it I think it's likely it will be left inconsistent as we have no burndown list of issues like this.

Reply:
Sure thing!  I've opened #10793 for this change.